src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -304,7 +310,13 @@ ReactDOMComponent.Mixin = {
 
     switch (this._tag) {
       case 'input':
-        ReactDOMInput.postMountWrapper(this, transaction, props);
+      case 'textarea':
+        if (props.autoFocus) {
```

Comment:
Just inlining it since it's the same for input/textarea/select and we don't have anything else like this anyway.